### PR TITLE
[ADF-4828] Should display the custom action menu for Process Cloud List

### PR DESCRIPTION
**Pull request by gmandakini**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/process-services-cloud/process-list-cloud-action-menu.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            await tasksCloudDemoPage.actionAdded('edit');
+            await tasksCloudDemoPage.addAction('delete');
+            await tasksCloudDemoPage.actionAdded('delete');
+            await tasksCloudDemoPage.clickAppButton();
+            await processCloudDemoPage.clickOnProcessFilters();
+            await processCloudDemoPage.runningProcessesFilter().clickProcessFilter();
+        });
+
+        it('[C315236] Should be able to see and execute custom action menu', async () => {
+            await expect(await processCloudDemoPage.getActiveFilterName()).toBe('Running Processes');
```
Comment (davidcanonieto) [inline e2e/process-services-cloud/process-list-cloud-action-menu.e2e.ts]: Do we need to wait for every single line of code?

Comment (gmandakini) [reply e2e/process-services-cloud/process-list-cloud-action-menu.e2e.ts]: I think so, not sure, what is an alternative to this?

**Comment by DenysVuika** [comment]
I have rebased this PR

File: lib/testing/src/lib/process-services-cloud/pages/process-list-cloud-component.page.ts [hunk 2]
```diff
@@ -100,4 +102,21 @@ export class ProcessListCloudComponentPage {
...
+        await BrowserVisibility.waitUntilElementIsVisible(this.actionMenu);
+        const actionButton = element(by.css(`button[data-automation-id*="${action}"]`));
+        await BrowserActions.click(actionButton);
+    }
+
+    async rightClickOnRow(processInstance: string): Promise<void> {
+        await this.dataTable.rightClickOnRow('Id', processInstance);
+    }
+
+    async pressContextMenuActionNamed(actionName): Promise<void> {
```
Comment (adinapitul) [inline lib/testing/src/lib/process-services-cloud/pages/process-list-cloud-component.page.ts]: Could use "click" instead of "press", consistency.

**Pull request merged**